test/deployment/000025-create-test-network.ts

```diff
@@ -1,13 +1,19 @@
 import { PendingWithdrawals, PoolCollection } from '../../components/Contracts';
 import { NetworkSettingsV1 } from '../../components/LegacyContractsV3';
-import { DeployedContracts, InstanceName, isLive } from '../../utils/Deploy';
+import { MAX_UINT256 } from '../../utils/Constants';
+import { DeployedContracts, InstanceName, isLive, TestTokenInstanceName } from '../../utils/Deploy';
 import { duration } from '../../utils/Time';
 import { TokenData, TokenSymbol } from '../../utils/TokenData';
-import { toPPM, toWei } from '../../utils/Types';
+import { min, toPPM, toWei } from '../../utils/Types';
 import { describeDeployment } from '../helpers/Deploy';
 import { expect } from 'chai';
+import { BigNumberish } from 'ethers';
 import { getNamedAccounts } from 'hardhat';
 
+const x = MAX_UINT256;
+console.log('AAA', x === MAX_UINT256);
+console.log('BBB', x.eq(MAX_UINT256));
```

Comment:
Is this on purpose?

Reply:
Yeah, wanted to demonstrate you that `===` works as well :)